nixos/tests/ctrld.nix

```
@@ -0,0 +1,21 @@
+
+import ./make-test-python.nix ({ pkgs, ... }: {
+  name = "ctrld";
+
+  meta = with pkgs.lib.maintainers; {
+    maintainers = [ arthsmn ];
+  };
+
+  nodes = {
+    client = { pkgs, ... }: {
+      environment.systemPackages = [ pkgs.dig ];
+      services.ctrld.enable = true;
+    };
+  };
+
+  # https://github.com/Control-D-Inc/ctrld#basic-run-mode
+  testScript = ''
+    client.wait_for_unit("ctrld")
+    client.wait_until_succeeds("dig verify.controld.com @client +short")
```

from ofborg:
```
> Exception: action timed out after 900 seconds
```

likely fix:

```suggestion
    client.wait_until_succeeds("dig verify.controld.com @127.0.0.1 +short")
```

For `client` to work, ctrld may need to be listening on `0.0.0.0`

---

Done.